.github/workflows/vast.yaml

```diff
@@ -655,13 +655,16 @@ jobs:
         working-directory: ${{ env.BUILD_DIR }}
         run: |
           ctest --parallel
-      - name: Install
-        run: |
-          cmake --install "$BUILD_DIR"
       - name: Run Integration Tests
         id: integration_tests
+        # We intentionally run the integration tests before installing
+        # them, because that is a use-case we want to explicitly support
+        # for easier plugin development.
```

Comment:
```suggestion
        # We intentionally run the integration tests before installing,
        # because that is a use-case we want to explicitly support for
        # easier plugin development.
```